lib-injection/auto_inject.rb

```diff
@@ -0,0 +1,56 @@
+if !ENV['skip_autoinject']
+  if system 'skip_autoinject=true bundle show ddtrace'
+    puts 'ddtrace already installed... skipping auto-injection'
+  else
+    version = "<DD_TRACE_VERSION_TO_BE_REPLACED>"
+    sha = "<DD_TRACE_SHA_TO_BE_REPLACED>"
+
+    condition =
+      if !version.empty?
+        # For public release
+        "--version '#{version.gsub(/^v/, '')}'"
```

Comment:
It's good to [escape strings before shell execution](https://ruby-doc.org/stdlib-2.5.1/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellescape) to avoid [Command Injection](https://owasp.org/www-community/attacks/Command_Injection) attacks:

```suggestion
        "--version '#{version.gsub(/^v/, '').shellescape}'"
```

Reply:
The same should be done for all strings interpolated into shell commands.